Resources/Help/analyses/anovarepeatedmeasures_nl.md

```diff
@@ -83,7 +83,7 @@ Sleep een of meer namen van factoren naar de rechter kolom om een post-hoc test
   - label y-as: Het label van de y-as kan handmatig worden bijgesteld.
 
 - weergeven:
-	- Geef foutmarges weer: Als u deze optie selecteert worden er foutmarges weergegeven in de grafiek. De foutmarges kunnen ofwel betrouwbaarheidsintervallen ofwel standaardfouten weergeven. Om precieze intervallen of standaard fouten te krijgen wordt de data genormaliseerd door de gemiddelde van elke observatie af te trekken en dan het totale gemiddelde aan elke observatie op te tellen. De varianties van de resulterende genormaliseerde waarden in elke conditie, en daardoor de grootte van de foutmarges, hangt niet langer af van de participant effecten en zijn zo een meer accurate representatie van de experimentele manipulatie. Zie Morw (2008) voor meer detail. 
+	- Geef foutmarges weer: Als u deze optie selecteert worden er foutmarges weergegeven in de grafiek. De foutmarges kunnen ofwel betrouwbaarheidsintervallen ofwel standaardfouten weergeven. Om precieze intervallen of standaardfouten te krijgen wordt de data genormaliseerd door de gemiddelde van elke observatie af te trekken en dan het totale gemiddelde aan elke observatie op te tellen. De varianties van de resulterende genormaliseerde waarden in elke conditie, en daardoor de grootte van de foutmarges, hangt niet langer af van de participant effecten en zijn zo een meer accurate representatie van de experimentele manipulatie. Zie Morw (2008) voor meer detail.
```

Comment:
het gemiddelde niet de.
en voor meer details?

Reply:
Scherp zeg!! Het is al een moeilijk tekst en als er dan ook nog grammos in staan. (variant op typo) ok